QUICKSTART.md

```diff
@@ -0,0 +1,81 @@
+# cms-l1t-analysis
```

Comment:
This looks like something that would be useful to integrate into the docs/ folder so it would appear on
http://cms-l1t-analysis.readthedocs.io/en/latest/